packages/tizen_rpc_port/tizen/src/common.hh

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef COMMON_HH
+#define COMMON_HH
+
+#include <flutter/event_channel.h>
+#include <flutter/standard_method_codec.h>
+#include <rpc-port-parcel.h>
+#include <rpc-port.h>
+
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+#include <vector>
+
+namespace tizen {
+
+typedef std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> EventSink;
+
+struct RpcPortResult {
+  RpcPortResult() : error_code(RPC_PORT_ERROR_NONE) {}
+  explicit RpcPortResult(int code) : error_code(code) {}
+
+  // Returns false on error
+  operator bool() const { return error_code == RPC_PORT_ERROR_NONE; }
+
+  const char* message() { return get_error_message(error_code); }
+
+  int error_code;
+};
+
+RpcPortResult CreateResult(int return_code);
+
+class RpcPort {
+ public:
+  enum Type {
+    Main,
+    Callback,
+  };
```

Comment:
`Type` is not used, I believe you meant to use this enum instead of `int32_t`.

Reply:
Yes, you are right. I removed it. Thanks :)

Reply:
> You may want to enable auto formatting in the VS Code settings:
> 
> * https://github.com/flutter-tizen/flutter-tizen/wiki/Style-guide
> * https://github.com/flutter-tizen/flutter-tizen/wiki/Recommended-settings-for-VS-Code

I performed the clang format.